core/spec.md

```diff
@@ -1713,39 +1709,43 @@ retrieve previous Versions of the Resource. In this respect, Resources have
 a 2-layered definition. The first layer is the Registry Resource entity itself,
 while the second layer is the `versions` collection - the version history of
 the Resource. The Resource entity can be thought of as an alias for the
-"latest" Version of the Resource, and as such, many of the attributes
-shown when retrieving the Resource will come from that Version. However,
-there are a few exceptions:
+"latest" Version of the Resource, and as such, most of the attributes
```

Comment:
Having removed "shown", we're missing a verb: "most of the attributes ??? when processing the resource".

Reply:
fixed